source/deployment/video-and-audio-calling.rst

```diff
@@ -17,6 +17,15 @@ Zoom
 - Docs: https://docs.mattermost.com/integrations/zoom.html
 - Source code: https://github.com/mattermost/mattermost-plugin-zoom
 
+Cisco Webex Cloud
+~~~~~~~~~~~~~~~~~~~~
+
+- Start and join voice calls, video calls and use screen sharing with your team members, developed by Mattermost.
+- Using a ``/webex`` command or clicking a video icon in a Mattermost channel invites team members to join a Webex call, hosted using the credentials of the user who initiated the call.
+- // ?? Requires 1 paid `Zoom Pro, Business, Education, or API plan, <https://zoom.us/pricing>`_ to generate an API key and secret for $14.99/month.
```

Comment:
```suggestion
- // ?? Requires a single paid `Zoom Pro, Business, Education, or API plan, <https://zoom.us/pricing>`_ to generate an API key and secret for $14.99/month.
```
Are all the plans $14.99/month? Or is the API key and secret $14.99/month?

Reply:
I suggest not putting any pricing in at all - we then need to keep it in synch with the company's pricing plans....